### PR TITLE
Add recipes repo link to readme

### PR DESCRIPTION
**Pull request by serinamarie**
@ahuang11 Not sure if diving deeper is the best word-choice here, but it worked well in recipes :)

### CODE REVIEW

File: {{cookiecutter.collection_name}}/README.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ def example_flow():
 example_flow()
 ```
 
+### Diving Deeper
+For more examples on how to use our collections and blocks, head to our [Prefect Recipes repo](https://github.com/PrefectHQ/prefect-recipes/)!
```
Comment (ahuang11) [inline {{cookiecutter.collection_name}}/README.md:68]:
```suggestion
### Diving deeper

For more examples on how to use our collections and blocks, head to our [Prefect Recipes repo](https://github.com/PrefectHQ/prefect-recipes/)!
```

**Review by ahuang11 — APPROVED** [review]

**Comment by ahuang11** [comment]
With this, we should probably add more collection related recipes to prefect-recipes since I think all of the ones on the recipes page at the moment is core?

**Comment by serinamarie** [comment]
At the top of my head I know we have prefect-snowflake, prefect-airbyte, and prefect-aws. I can make a recipe request for more frequently used collections. Any in particular you were thinking of?

**Comment by desertaxle** [comment]
> At the top of my head I know we have prefect-snowflake, prefect-airbyte, and prefect-aws. I can make a recipe request for more frequently used collections. Any in particular you were thinking of?

Based on a scan of the recipes repo, those recipes are for 1.0. I don't think that we have any recipes that use collections, which is something we should fix! I think the link to the Prefect Recipes repo can be added on a case by case basis once a recipe for the given collection exists.

**Comment by serinamarie** [comment]
Maybe I'm misunderstanding something. Here are some recipes that utilize the prefect-aws, prefect-airbyte, and prefect-snowflake collections.

[Export Airbyte Configuration and Load to S3 bucket using blocks, including Python-based deployment](https://github.com/PrefectHQ/prefect-recipes/blob/main/flows-advanced/etl/export-airbyte-config-and-write-to-s3-bucket-using-blocks.py)

[ELT with Snowflake Using Async and Blocks](https://github.com/PrefectHQ/prefect-recipes/blob/main/flows-advanced/etl/elt-with-snowflake.py)

**Comment by desertaxle** [comment]
Ah, I missed those! I think it makes sense to have a link to Prefect Recipes repo in the collection repos that correspond to those recipes. Potentially even link to those specific recipes in addition to a link to the recipes repo landing page.

**Comment by ahuang11** [comment]
Are there plans to make a prefect-recipes webpage, combining visual cards like https://orion-docs.prefect.io/recipes/recipes/ and filtering ability like https://www.prefectlabs.com/experts? Then all collections can link to a specific URL with the filter applied corresponding to the collection, e.g. `Click [here](https://www.prefectlabs.com/recipes?tag=prefect-snowflake) to see recipes related to the collection`

I imagine the single source of truth / backend data can be the `prefect-recipes` page with addition of collection tags.

I feel like if we add a link to those specific recipes, there will be three places that we need to keep track of, which is likely to become out of sync:
1. [prefect-recipes](https://github.com/PrefectHQ/prefect-recipes)
2. [Prefect docs](https://orion-docs.prefect.io/recipes/recipes/#prefect-recipes)
3. the corresponding Collection

**Comment by serinamarie** [comment]
> I feel like if we add a link to those specific recipes to each repo, there will be three places that we need to keep track of, which is likely to become out of sync:
> 
> 1. [prefect-recipes](https://github.com/PrefectHQ/prefect-recipes)
> 2. [Prefect docs](https://orion-docs.prefect.io/recipes/recipes/#prefect-recipes)
> 3. the corresponding Collection

Indeed, I think we should limit the sources of truth. But I think it's a good idea to discuss at the next recipes meeting, so thanks for that!

**Comment by ahuang11** [comment]
Closing this for now, opened https://github.com/PrefectHQ/prefect-recipes/issues/155 to track.

**Pull request closed without merging**